### ISSUE TITLE
Add docker file

### ISSUE THREAD

**Issue by hgducharme** [hgducharme/meatball#45]

**Comment by hgducharme** [comment]
Closed by #50

**Comment by hgducharme** [comment]
Ok I think i'm a bit off base here with docker *inside* the makefile. Maybe I should just make a directory called `docker/` and have shell scripts in there like `build.sh`, `cppcheck.sh` etc, and then we can run from the command line: 

```
$ ./docker/build.sh
$ ./docker/cppcheck.sh
$ ./docker/linter.sh
```

etc.

**Comment by hgducharme** [comment]
Nevermind, closed by #53. Will be deleting the #50 merge request soon.
